android/app/src/main/java/org/uwaterloo/subletr/pages/createlisting/CreateListingPageView.kt

```diff
@@ -331,7 +272,143 @@ fun CreateListingPageView(
+				)
+
+				Row(
+					modifier = Modifier.fillMaxWidth(),
+					horizontalArrangement =  Arrangement.spacedBy(dimensionResource(id = R.dimen.s)),
+					verticalAlignment = Alignment.CenterVertically,
+				) {
+					CreateListingNumericalInputs(
+						modifier = Modifier.weight(1f),
+						labelId = R.string.num_bedrooms,
+						viewModelStream = viewModel.numBedroomsStream,
+						uiStateValue = uiState.numBedrooms,
+					)
+					CreateListingNumericalInputs(
+						modifier = Modifier.weight(1f),
+						labelId = R.string.bedrooms_in_unit,
+						viewModelStream = viewModel.totalNumBedroomsStream,
+						uiStateValue = uiState.totalNumBedrooms,
+					)
+				}
+
+				Spacer(
+					modifier = Modifier.height(dimensionResource(id = R.dimen.m)),
+				)
+
+				Row(
+					modifier = Modifier.fillMaxWidth(),
+					verticalAlignment = Alignment.CenterVertically,
+					horizontalArrangement = Arrangement.SpaceBetween,
+				) {
+					Text(
+						text = stringResource(id = R.string.bathrooms),
+						color = MaterialTheme.subletrPalette.primaryTextColor,
+						style = SubletrTypography.displaySmall,
+					)
+				}
+
+				Spacer(
+					modifier = Modifier.height(dimensionResource(id = R.dimen.xs)),
+				)
+
+				Row(
+					modifier = Modifier.fillMaxWidth(),
+					horizontalArrangement =  Arrangement.spacedBy(dimensionResource(id = R.dimen.s)),
+					verticalAlignment = Alignment.CenterVertically,
+				) {
+					CreateListingNumericalInputs(
+						modifier = Modifier.weight(1f),
+						labelId = R.string.num_bathrooms,
+						viewModelStream = viewModel.numBathroomsStream,
+						uiStateValue = uiState.numBathrooms,
+					)
+					CreateListingNumericalInputs(
+						modifier = Modifier.weight(1f),
+						labelId = R.string.bathrooms_in_unit,
+						viewModelStream = viewModel.totalNumBathroomsStream,
+						uiStateValue = uiState.totalNumBathrooms,
+					)
+				}
+
+				Spacer(
+					modifier = Modifier.height(dimensionResource(id = R.dimen.s)),
+				)
+
+				CreateListingSelectionInput(
+					dropdownItems = EnsuiteBathroomOption.values(),
+					labelId = R.string.ensuite_bathroom,
+					selectedDropdownItem = uiState.bathroomsEnsuite,
+					dropdownItemToString = { stringResource(id = it.stringId) },
+					setSelectedDropdownItem = {
+						viewModel.bathroomsEnsuiteStream.onNext(it)
+					},
+				)
+
+				Spacer(
+					modifier = Modifier.height(dimensionResource(id = R.dimen.m)),
+				)
+
+				Row(
+					modifier = Modifier.fillMaxWidth(),
+					verticalAlignment = Alignment.CenterVertically,
+					horizontalArrangement = Arrangement.SpaceBetween,
+				) {
+					Text(
+						text = stringResource(id = R.string.additional_information),
+						color = MaterialTheme.subletrPalette.primaryTextColor,
+						style = SubletrTypography.displaySmall,
+					)
+				}
+
+				Spacer(
+					modifier = Modifier.height(dimensionResource(id = R.dimen.xs)),
+				)
+
+				CreateListingSelectionInput(
+					dropdownItems = Gender.values(),
```

Comment:
so there doesnt need to be all the gender options, this field is only for what gender they are looking to sublet to, so the options would be Male, Female, Any. Having prefer not to say or other dont make sense as options for who to sublet to

Reply:
right right, makes sense